### PR TITLE
feat: #120 static pages privacy policy and acceptable use policy

### PR DESCRIPTION
**Pull request by Anok666**

### CODE REVIEW

**Comment by agpul** [comment]
Add the dark blue bar with "Go to Dashboard" link

**Comment by Anok666** [comment]
> Add the dark blue bar with "Go to Dashboard" link

The dark blue bar with "Go to Dashboard" link was added. Plus strong style to both headers was added. Check it please.

**Comment by Marcelinna** [comment]
@Anok666 Can you use style as here https://search.marketplace.eosc-portal.eu/trainings/1000001
image


PR - blue bar is too wide and the font style should be changed
image

**Comment by eldenkat** [comment]
Checked the content, there are some minor amendments (commented in both google docs). Also have added one sentence in Privacy policy (marked in red) - _Recommended resources are provided via EOSC Marketplace website_ - to empahasise a connection between UD and resources catalogue.

File: ui/apps/ui/src/app/pages/acceptable-use-policy/acceptable-use-policy.component.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      <li>You shall only use the Services for lawful purposes and not breach, attempt to breach, nor circumvent administrative or security controls.</li>
+      <li>You shall respect intellectual property and confidentiality agreements.</li>
+      <li>You shall protect your access credentials (e.g. passwords, private keys or multi-factor tokens); no intentional sharing is permitted.</li>
+      <li>You shall keep your registered information correct and up to date.</li>
+      <li>You shall promptly report known or suspected security breaches, credential compromise or misuse to the security contact stated below; and report any compromised credentials to the relevant issuing authorities.</li>
+      <li>Reliance on the Services shall only be to the extent specified by any applicable service level agreements listed below. Use without such agreements is at your own risk.</li>
+      <li>Your personal data will be processed in accordance with the <a href="https://docs.google.com/document/d/1Swi5XimmXH2XYHTZTXS4QU3oXUYlCP_ffqFhjydvxv0/edit">Privacy Policy of the EOSC User Dashboard</a>.</li>
```
Comment (michal-szostak) [inline ui/apps/ui/src/app/pages/acceptable-use-policy/acceptable-use-policy.component.html:18]: do we really want to pass google docs here?

**Review by michal-szostak — APPROVED** [review]
Just one question about those links to google docs - is it alright? @Marcelinna , @agpul ? @aleksandrahojszyk ?

File: ui/apps/ui/src/app/pages/acceptable-use-policy/acceptable-use-policy.component.html (hunk 1 above)
Comment (Anok666) [reply ui/apps/ui/src/app/pages/acceptable-use-policy/acceptable-use-policy.component.html]: no, google docs replaced by angular routing to privacy-policy

**Review by Marcelinna — APPROVED** [review]

**Review by eldenkat — APPROVED** [review]

**Pull request merged**